packages/components/src/LightBox/LightBox.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Button } from "@jobber/components/Button";
+import { LightBox } from "@jobber/components/LightBox";
+import { ExampleWithHooks } from "mdxUtils/ExampleWithHooks";
+
+<Meta title="Components/LightBox" component={LightBox} />
+
+# Light Box
+
```

Comment:
This seems to be missing the other styling when the lightbox is open
Storybook:
<img width="1779" alt="image" src="https://user-images.githubusercontent.com/17303106/220746380-5a59a7ad-b9de-4af1-b267-e53a0f38ae80.png">


Docs currently:

![image](https://user-images.githubusercontent.com/17303106/220746604-14f6eaf6-abe2-44b4-b9dd-4da1ef5f64f0.png)

Reply:
Also doesn't show on the canvas tab

Reply:
I knew I'd have nightmares with this

Reply:
This looks like an issue with the library `react-image-lightbox`, but it's already deprecated and it's been a while since the last update.

I tried upgrading it to `5.1.4` because they fixed [this issue](https://github.com/frontend-collective/react-image-lightbox/issues/136) that is related to the lightbox being inside an iFrame (our case), but no success :(

Reply:
Are we able to use the library that is recommended instead?
https://www.npmjs.com/package/react-photoswipe-gallery

We may want to add it to our list of packages to upgrade because it is no longer maintained.

I feel like this should be a blocker but I will defer to @chris-at-jobber

Reply:
Yeah this is a pretty bad representation of the component and this will also impact viewers of Gallery as that component consumes Lightbox

Reply:
If we can split it out to solve independently and land the rest of these docs, that could be OK for now. Would really like InputText to be available 😄

Reply:
I removed LightBox from this PR so we can land the other ones.